src/main/java/homework_3/Main.java

```diff
@@ -0,0 +1,26 @@
+package homework_3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
```

Comment:
Result of Main:
![image](https://user-images.githubusercontent.com/58872509/134149252-61309b68-0509-4f08-9ddb-ad05b048bf70.png)